### ISSUE TITLE
False Positive  in micrometer-core-1.0.5.jar

### ISSUE THREAD

**Issue by yaro24** [dependency-check/DependencyCheck#1565]
False positive on library 
micrometer-core-1.0.5.jar (part of org.springframework.boot:spring-boot-starter-actuator:jar:2.0.3.RELEASE)
Was reported as cpe:/a:git:git:1.0.5 
```xml
<!-- https://mvnrepository.com/artifact/io.micrometer/micrometer-core -->
<dependency>
    <groupId>io.micrometer</groupId>
    <artifactId>micrometer-core</artifactId>
    <version>1.0.5</version>
</dependency>
```

**Comment by electrified** [comment]
We are also seeing this.

**Comment by jeremylong** [comment]
Thanks - I will sweep through the FP before the next release.

**Comment by jeremylong** [comment]
This has already been fixed in the 5.0.0-M2 milestone release.
